src/server/blockchain/__tests__/adminWallet.test.js

```diff
@@ -48,7 +48,7 @@ describe('adminwallet', () => {
     const promises = []
     for (let i = 0; i < 5; i++) {
       const unverifiedAddress = generateWalletAddress()
-      // console.log('unverifiedAddress', unverifiedAddress)
+      const tx = await AdminWallet.whitelistUser(unverifiedAddress, 'did:gd')
```

Comment:
whats this for?

Reply:
I cannot make topWallet without adding to whitelistUser

Reply:
well then obviously its a broken behaviour, the tests are here for THIS PURPOSE EXACTLY

Reply:
this should work without whitelist
@AlexeyKosinski

Reply:
so we need to change smartcontract
because topWallet dodn't work without whitelist

Reply:
also need to add `did` param to AdminWallet.whitelistUser